src/plugins/kibana_usage_collection/server/collectors/management/schema.ts

```diff
@@ -11,120 +11,405 @@ import { UsageStats } from './types';
+    _meta: { description: 'timelion:graphite was changed.' },
+  },
+  'xpackDashboardMode:roles': {
+    type: 'keyword',
+    _meta: { description: 'xpackDashboardMode:roles was changed.' },
+  },
+  'securitySolution:ipReputationLinks': {
+    type: 'keyword',
+    _meta: { description: 'securitySolution:ipReputationLinks was changed.' },
+  },
   // non-sensitive
-  'visualize:enableLabs': { type: 'boolean' },
-  'visualization:heatmap:maxBuckets': { type: 'long' },
-  'visualization:colorMapping': { type: 'text' },
-  'visualization:regionmap:showWarnings': { type: 'boolean' },
-  'visualization:dimmingOpacity': { type: 'float' },
-  'visualization:tileMap:maxPrecision': { type: 'long' },
-  'csv:separator': { type: 'keyword' },
-  'visualization:tileMap:WMSdefaults': { type: 'text' },
-  'timelion:target_buckets': { type: 'long' },
-  'timelion:max_buckets': { type: 'long' },
-  'timelion:es.timefield': { type: 'keyword' },
-  'timelion:min_interval': { type: 'keyword' },
-  'timelion:default_rows': { type: 'long' },
-  'timelion:default_columns': { type: 'long' },
-  'timelion:es.default_index': { type: 'keyword' },
-  'timelion:showTutorial': { type: 'boolean' },
-  'securitySolution:timeDefaults': { type: 'keyword' },
-  'securitySolution:defaultAnomalyScore': { type: 'long' },
-  'securitySolution:refreshIntervalDefaults': { type: 'keyword' },
-  'securitySolution:enableNewsFeed': { type: 'boolean' },
-  'search:includeFrozen': { type: 'boolean' },
-  'courier:maxConcurrentShardRequests': { type: 'long' },
-  'courier:batchSearches': { type: 'boolean' },
-  'courier:setRequestPreference': { type: 'keyword' },
-  'courier:customRequestPreference': { type: 'keyword' },
-  'courier:ignoreFilterIfFieldNotInIndex': { type: 'boolean' },
-  'rollups:enableIndexPatterns': { type: 'boolean' },
-  'notifications:lifetime:warning': { type: 'long' },
-  'notifications:lifetime:banner': { type: 'long' },
-  'notifications:lifetime:info': { type: 'long' },
-  'notifications:lifetime:error': { type: 'long' },
-  'doc_table:highlight': { type: 'boolean' },
-  'discover:searchOnPageLoad': { type: 'boolean' },
+  'visualize:enableLabs': {
+    type: 'boolean',
+    _meta: { description: 'Non-default value of visualize:enableLabs' },
+  },
+  'visualization:heatmap:maxBuckets': {
+    type: 'long',
+    _meta: { description: 'Non-default value of visualivisualization:heatmapng' },
+  },
+  'visualization:colorMapping': {
+    type: 'text',
+    _meta: { description: 'Non-default value of visualization:colorMapping' },
+  },
+  'visualization:regionmap:showWarnings': {
+    type: 'boolean',
+    _meta: { description: 'Non-default value of visualization:regionmap' },
+  },
+  'visualization:dimmingOpacity': {
+    type: 'float',
+    _meta: { description: 'Non-default value of visualization:dimmingOpacity' },
+  },
+  'visualization:tileMap:maxPrecision': {
+    type: 'long',
+    _meta: { description: 'Non-default value of visualization:tileMap' },
```

Comment:
It looks like the props that had 2 `:` in their names are missing the last bit :)

```suggestion
    _meta: { description: 'Non-default value of visualization:tileMap:maxPrecision' },
```